templates/vue/cypress/integration/import-export.spec.js

```diff
@@ -1,5 +1,43 @@
 describe("Import Export", () => {
-  it("should be able to import a Tapestry using the file input", () => {
+  it("export endpoint should return not null response body", () => {
+    cy.fixture("one-node.json").as("oneNode")
+    cy.setup("@oneNode")
+    cy.server()
+    cy.route("GET", "**/tapestries/**/export").as("export")
+
+    cy.get('.settings-button').click()
+    cy.contains(/advanced/i).click()
+    cy.get('#export-button').click()
+
+    cy.wait('@export')
+    .then((xhr) => {
+        assert.isNotNull(xhr.response.body, "response body not null")
+    })
```

Comment:
I'd suggest to use the Cypress assertion for this to keep it consistent. You can do something like:

```suggestion
    cy.wait('@export').its("response.body").should("not.be.null")
```